convert.py

```diff
@@ -939,7 +1013,8 @@ def convert_to_output_type(model: LazyModel, output_type: GGMLFileType) -> LazyM
             for (name, tensor) in model.items()}
 
 def convert_model_names(model: LazyModel, params: Params) -> LazyModel:
-    tmap = gguf.get_tensor_name_map(ARCH, params.n_layer)
+    tmap = gguf.TensorNameMap(ARCH, params.n_layer)
+    should_skip: Set[gguf.MODEL_TENSOR] = gguf.MODEL_TENSOR_SKIP.get(ARCH, set())
```

Comment:
This is typed as 'Set' here, but MODEL_TENSOR_SKIP is typed as `Dict[MODEL_ARCH, List[MODEL_TENSOR]]`.

Reply:
Good call and appreciate all the feedback. I changed directions with how that got implemented and forgot to make the changes.

I'm about to go to bed but I will go through all of these tomorrow hopefully.